documentation/markup.rst

```diff
@@ -678,8 +676,12 @@ Roles
 As said before, Sphinx uses interpreted text roles to insert semantic markup in
 documents.
 
-Names of local variables, such as function/method arguments, are an exception,
-they should be marked simply with ``*var*``.
+There are a couple of cases where roles can be omitted in favor of
+simpler markup:
+
+* Function and method arguments can be marked with ``*arg*``.
+* ``True``, ``False``, ``None``, and variables can be marked with
+  :literal:`\`\`...```.
```

Comment:
```suggestion

* Function and method arguments should be written as ``*arg*``
  (rendering as *arg*)
* ``True``, ``False``, and ``None`` should be written with :literal:`\`\`...```
  (rather than ``:const:``
```

* Show rendered form for *arg*
* Use clearer and more idiomatic wording/phrasing
* Make this at least a SHOULD, not just a MAY
* Explicitly specify the alternative that is being discouraged here
* Fix grammar

Also, after a lot of thought, I suggest eliding the line above specifically mentioning how arguments in the CPython documentation are written, to separate the general reST material in the previous top-level section (with a few CPython-specific bits that can be factored out) from the guidance specific to CPython, and avoiding duplication of the same information

Reply:
> * Show rendered form for arg
> * Use clearer and more idiomatic wording/phrasing

ok.

> * Make this at least a SHOULD, not just a MAY

Agreed :+1: 

> * Explicitly specify the alternative that is being discouraged here

I would leave this out, both because there isn't a clear alternative (some people use `:const:`, some `:keyword:`) and because mentioning doesn't seem useful.

> separate the general reST material in the previous top-level section (with a few CPython-specific bits that can be factored out) from the guidance specific to CPython

Do you think this is a separate issue, or should be tackled in this PR?

Reply:
> I would leave this out, both because there isn't a clear alternative (some people use :const:, some :keyword:) and because mentioning doesn't seem useful.

Okay, sure. I updated my above suggestion accordingly.

> Do you think this is a separate issue, or should be tackled in this PR?

Separate, since its a substantially bigger change that should be discussed first and IMO out of scope here (the sentence in particular was added with this PR, I just wasn't viewing the right commit to be able to change it). I brought it up to put that suggestion into context with my big-picture view on this.

Reply:
Any reason for omitting variables from the list of elements that should use ` ``...`` `?

Reply:
Separation of concerns; they are already listed above in the general Markdown inline syntax section, along with a number of other similar elements that should use ``...``, so it doesn't make sense to repeat them here without repeating everything. This section specifically discusses the additional specific markup usage particular to CPython, rather than being a general list of places ``...`` should be used